target/linux/ath79/dts/qca9558_tplink_tl-wpa8730-v1.dts

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
```

Comment:
This include should be moved in front of the others (just have a look at recently changed files)

Reply:
done